OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -187,7 +196,7 @@ public Aircraft(ActorInitializer init, AircraftInfo info)
 
 			// TODO: HACK: This is a hack until we can properly distinguish between airplane and helicopter!
 			// Or until the activities get unified enough so that it doesn't matter.
-			IsPlane = !info.CanHover;
+			IsPlane = !info.VTOL;
```

Comment:
Well, I went ahead and removed `IsPlane` afterall, it's not that much more work and it won't be much more work to shift the `if`s from the IMove activities to the merged activities later.